css/style.css

```diff
@@ -0,0 +1,228 @@
+    background-color: white;
+}
+
+/* body end*/
+
+/* header */
+
+header {
+    margin: 0;
+    padding: 0;
+    top: 0;
+    position: fixed;
+    width: 100%; 
+    height: 10rem;
+    background-color: rgb(56, 49, 49);
+}
+
+.logo-contact {
+    width: 100%;
+    top: 0;
+    height: 5rem; 
+}
+
+.contact {
+    text-align: right;
+    margin-right: 2rem;
+    padding-bottom: 5rem;
+}
+
+.contact span {
+    background-color: rgb(22, 21, 21);
+    padding: 0.2rem;
+    border: 0.15rem solid rgb(29, 26, 24);
+    border-radius: 1rem;
+}
+
+.contact span a {
+    color: rgb(240, 24, 114);
+}
+
+.contact span a:hover {
+    color: rgb(235, 214, 27);
+}
+
+.logo-contact img {
+    width: 5%;
+    height: auto;
+    padding: 0.5rem;
+}
+
+.logo-contact a {
+    text-decoration: none;
+    font-size: 1.5rem;
+}
+
+.header-nav-list {
+    width: 100%; 
+    height: 5rem;
+    padding: 0.01rem;
+    background-color: rgb(165, 116, 42);
+}
+
+.header-nav-list li {
+    float: left;
+    list-style: none;
+    text-align: center;
+}
+
+.header-nav-list a {
+    color: brown;
+    font-size: 1.5rem;
+    text-decoration: none;
+    padding-right: 2rem;
+}
+
+.header-nav-list a:hover {
+    color: rgb(36, 35, 35);
+}
+/* header end */
+
+/* main */
+
+main {
+    margin: 5rem;
+    padding: 0;
+    width: 100%;
+    height: auto;
+}
+
+.section1 {
+    width: 80%;
+    padding: 5rem;;
+    height: auto;
+    display: flex;
+    justify-content: flex-start;
+}
+
+.div1{
+    background-color: rgb(235, 221, 231);
+    width: 40%;
+    margin-left: 0;
+    position: relative;
+}
+
+.div2 {
+    width: 50%;
+   
+}
+
+.frame {
+    width: 100%;
+    top: 0;
+    margin: 0;
+    padding: 0;
+}
+
+.div2 .frame ul {
+    position: relative;
+    list-style: none;
+    padding: 0;
+    width: 100%;
+    height: 10%;
+}
+
+.div2 .frame li {
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: auto;
+    opacity: 0;   
+}
+
+.div2 .frame li:first-child {
+    opacity: 1;
+}
+
+.div2 .frame li img {
+    width: 100%;
+    height: 40%;
+    object-fit: cover;
+}
+
+.slides {
+    width: 100%;
+    margin-top: 5%;
+    padding-top: 80%;
+    display: flex;
+}
+
+.slides a {
+    border: 2px solid rgb(151, 33, 33);
+} 
+
+.slides a:hover {
+    border: 2px solid blue;
+}
+
+.slides a img {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+}
+
+.div2 .frame li:target {
+    opacity: 1;
+}
+
+.section2 {
+    width: 80%;
+    display: flex;
+    justify-content: center;
+    padding: 50px;
+    background-image: url(https://img.glyphs.co/img?q=85&w=900&src=aHR0cHM6Ly9zMy5tZWRpYWxvb3QuY29tL2Jsb2ctaW1hZ2VzL1NUUEItSW1hZ2UtMDguanBnP210aW1lPTIwMTgwOTE0MTAwNjQx);
+    
+}
+
+.section2 img {
+    width: 20%;
+    height: auto;
+    border: 2px solid beige;
+    padding-right: 10px;
+}
+/* main end */
+
+/* footer */
+
+footer {
+    padding: 2rem;
+    width: 100%;
+    background-color: rgb(83, 80, 80);
+}
+
+footer nav li {
+    float: left;
```

Comment:
I would probably change this to display: inline-block or explore using flexbox for these footer links. When you float the footer can't tell what height it need to be and so the links break the bounds of the footer so to speak